rebar.config

```diff
@@ -27,7 +27,7 @@
         {parse_trans, "3.2.0"},
         {mnesia_rocksdb, ".*", {git, "https://github.com/aeternity/mnesia_rocksdb.git", {ref,"618870f"}}},
         {enacl, ".*", {git,"https://github.com/aeternity/enacl.git", {ref,"2f50ba6"}}},
-        {enoise, ".*", {git, "https://github.com/aeternity/enoise.git", {ref, "e3baa21"}}},
+        {enoise, ".*", {git, "https://github.com/aeternity/enoise.git", {ref, "7347906"}}},
```

Comment:
Wrt https://github.com/aeternity/enoise/compare/e3baa21...7347906#diff-93af256510f79796918e0ff06b07fbb2R109
I see it relies on being linked but when controlling process changes who does the new linking? (I did not arrive at checking yet.)

Reply:
This is the safety mechanism monitoring from the inside. I had envisioned the linking to be handled by the controlling process (automatically on accept/connect and later by hand if giving away the control) - but of course you could think of it being done automatically?